crates/nu-protocol/src/signature.rs

```diff
@@ -113,6 +113,42 @@ impl std::fmt::Display for Category {
     }
 }
 
+pub fn category_from_string(category: &str) -> Category {
+    match category {
+        "bits" => Category::Bits,
+        "bytes" => Category::Bytes,
+        "chart" => Category::Chart,
+        "conversions" => Category::Conversions,
+        "core" => Category::Core,
```

Comment:
As `Category::Core` is basically just the `nu-cmd-lang` keywords (and for some reason the `commandline` variants in `nu-cli` that we may want to move over) I think we could reserve the rights that only Nushell itself can declare something as `core`

Reply:
Yeah removing the line would fall through to `Custom("core")` and then would be Displayed as `core` again mhh.

Reply:
I'll make it "custom_core" but this also will prevent us from making stdlib commands "core", which may be ok. Not sure.